### PR TITLE
Update README.md and VSC Marketplace

### PR DESCRIPTION
**Pull request by iskyOS**
Updates the README:
-  "Commands" -> "Prompts"

Updates VSC Marketplace
- 

## Test plan

Marketing copy only.

### CODE REVIEW

File: vscode/README.md [hunk 1]
```diff
@@ -23,23 +23,21 @@ For example, you can ask Cody:
...
-
-<img src="https://storage.googleapis.com/sourcegraph-assets/blog/vs-code-onboarding-walkthrough-dec-2023-convert-html-to-md.gif" width="480" alt="Custom command">
+You can also create your own prompts and save them in the [Prompt Library](https://sourcegraph.com/docs/cody/capabilities/commands#prompt-library) to tailor Cody to your workflow. 
 
 ## Choose Your LLM
 
-Cody Pro users can now select the LLM they want to use for chat and experiment to choose the best model for the job. Choose from Claude 3 Opus, Claude 3.5 Sonnet, Claude 3 Sonnet, Claude 3 Haiku, ChatGPT 4o, ChatGPT 4 Turbo, ChatGPT 3.5 Turbo, Google Gemini 1.5 Pro, Gemini 1.5 Flash, and Mixtral.
+Cody users can select the LLM they want to use for chat and experiment to choose the best model for the job. Choose from Claude 3 Opus, Claude 3.5 Sonnet, Claude 3 Haiku, GPT-4 Turbo, GPT-3.5 Turbo, Google Gemini 1.5 Pro, Gemini 1.5 Flash, and Mixtral. Cody Pro users can also select Claude 3 Opus and GPT-4o.
```
Comment (ichim-david) [inline vscode/README.md:40]: @iskyOS Nowadays as non-enterprise customers have a shorter list of models to choose from
models-available
Expect the o1 family for which I have access I believe that this is the current list of available models so no GPT-4 Turbo or GPT-3-5.

https://sourcegraph.com/docs/cody/capabilities/supported-models#chat-and-commands
If you look here https://github.com/sourcegraph/cody/blob/main/lib/shared/src/models/dotcom.ts#L35
Free users have access to 
Sonnet claude-3.5
Claude 3 Haiku
gemini-1.5-pro
gemini-1.5-flash

PRO users also get
Claude 3 Opus
Chat GPT4-o

Comment (iskyOS) [reply vscode/README.md:40]: Thanks @ichim-david. Updating:

```suggestion
Cody users can select the LLM they want to use for chat and experiment to choose the best model for the job. Choose from Claude 3.5 Sonnet, Claude 3 Haiku, Gemini 1.5 Pro, Gemini 1.5 Flash, and Mixtral 8x7B. Cody Pro users can also select Claude 3 Opus and GPT-4o.
```

**Review by kalanchan — APPROVED** [review]
nice!

File: vscode/README.md (hunk 1 above)
Comment (jdorfman) [reply vscode/README.md]: Thanks @ichim-david!

**Pull request merged**